js/script.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        setTimeout(function () {
+            elements.cursorPosition = elements.textArea.prop("selectionStart");
+            if(elements.cursorPosition){
+                let value = elements.textArea.val();
+                elements.startPos = getStartPosition(value, elements.cursorPosition-1);
+                if(elements.startPos){
+                    let text = value.substring(elements.startPos, elements.cursorPosition);
+                      return  find(text, users, elements);
+                }
+                return elements.showUsers.empty();
+                }
+        }, 0);
+
+    }
+
+    function getStartPosition(str, pos) {
+        if (!str.substring(pos, pos + 1).match(/[A-Za-z]/)) {
+            return str.substring(pos, pos + 1) === '@' ? pos + 1: null;
+        } else if (pos === 0) {
+            return 0
+        } else {
+            return getStartPosition(str, pos - 1);
+        }
+    }
+
+    function find(user, users, elements) {
+        elements.showUsers.empty();
+        user = user.toLowerCase();
+        for (let i = 0; i < users.length; i++) {
```

Comment:
@lianatumikyan if you want to get first 10 items of the array you can use slice method